CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -129,16 +129,16 @@ class HomeTableViewCell: UITableViewCell {
                 make.top.equalTo(sectionLabel.snp.bottom).offset(12)
                 make.leading.equalTo(16)
                 make.height.equalTo(24)
-                make.trailing.equalTo(container.snp.centerX).inset(9)
-                make.bottom.equalTo(container).inset(16)
+                make.trailing.equalTo(containerView.snp.centerX).inset(9)
+                make.bottom.equalTo(containerView).inset(16)
             }
         } else {
             removeButton.snp.remakeConstraints { make in
                 make.top.equalTo(sectionLabel.snp.bottom).offset(12)
                 make.leading.equalTo(16)
                 make.height.equalTo(24)
-                make.trailing.equalTo(container).inset(16)
-                make.bottom.equalTo(container).inset(16)
+                make.trailing.equalTo(containerView).inset(16)
+                make.bottom.equalTo(containerView).inset(16)
```

Comment:
same about `superview` and you can also combine these two together: `make.trailing.bottom.equalToSuperview().inset(16)`